src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdApi.java

```diff
@@ -387,6 +387,7 @@ public CaseDetails submitEventForAttachScannedDocs(
             );
         } catch (FeignException e) {
             debugCcdException(log, e, "Failed to call 'submitEventForCaseWorker'");
+            log.error("CCD submitEventForCaseWorker for caseRef: {} ", caseRef, e);
```

Comment:
This exception is rethrown, is it not logged by an error handler?

Reply:
would this log expose sensitive information ?